src/games/strategy/triplea/ui/TripleAMenu.java

```diff
@@ -988,8 +988,8 @@ private void createAndSaveStats(final boolean showPhaseStats) {
       }
       text.append("\n");
       clone.getHistory().gotoNode(clone.getHistory().getLastNode());
-      @SuppressWarnings("rawtypes")
-      final Enumeration nodes = ((DefaultMutableTreeNode) clone.getHistory().getRoot()).preorderEnumeration();
+      @SuppressWarnings("unchecked")
+      final Enumeration<HistoryNode> nodes = ((DefaultMutableTreeNode) clone.getHistory().getRoot()).preorderEnumeration();
```

Comment:
please remove the unnecessary casts, it's presumably the reason for this whole exercise.